src/test/java/com/EighthLight/app/TestPlayer.java

```diff
@@ -1,14 +1,69 @@
 package com.EighthLight.app;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
 import static org.junit.jupiter.api.Assertions.*;
 
 class TestPlayer {
+    private ArrayList gameIsTiedValues = new ArrayList();
+    private ArrayList aPlayerWonValues = new ArrayList();
+    private ArrayList boardState = new ArrayList();
+    private String correctInput;
+    private String incorrectInput;
+
+    @BeforeEach
+    private void setUp() {
+        correctInput = Constants.CORRECT_INPUT;
+        incorrectInput = Constants.INCORRECT_INPUT;
+        gameIsTiedValues.add(false);
+        aPlayerWonValues.add(false);
+    }
 
     @Test
     void getPlayerSymbolReturnsThePlayersSymbol() {
-        Player playerOne = new Player("X");
+        ArrayList userInputs = new ArrayList(Arrays.asList(correctInput));
+        MockUi ui = new MockUi(userInputs);
+
+        Player playerOne = new Player("X", ui);
 
         assertEquals("X", playerOne.getSymbol());
     }
+
+    @Test
+    void makeMoveWithCorrectInput() {
```

Comment:
maybe rename this to `testExpectedMethodsAreCalledWhenMakeMoveIsExecutedWithCorrectArgs`